pkg/cmd/server/admin/create_keypair.go

```diff
@@ -39,8 +40,8 @@ var createKeyPairLong = templates.LongDesc(`
 	    %[1]s --public-key=$CONFIG/serviceaccounts.public.key --private-key=$CONFIG/serviceaccounts.private.key
 	`)
 
-func NewCommandCreateKeyPair(commandName string, fullName string, out io.Writer) *cobra.Command {
-	options := &CreateKeyPairOptions{Output: out}
+func NewCommandCreateKeyPair(commandName string, fullName string, streams genericclioptions.IOStreams) *cobra.Command {
+	options := &CreateKeyPairOptions{Output: streams.Out}
```

Comment:
Same thing here, can we wire IOStreams through the options struct?
Also, to keep consistency, create a constructor method `NewCreateKeyPairOptions(genericclioptions.IOStreams) *CreateKeyPairOptions`

Reply:
I don't want to deal with cleaning the commands now, I'm just cleaning the IOStreams. We'll be going through the commands later on.